TLM/TLM/Patch/_CitizenAI/_ResidentAI/GetVehicleInfoPatch.cs

```diff
@@ -126,15 +128,36 @@ public class GetVehicleInfoPatch {
             // NON-STOCK CODE START
             VehicleInfo carInfo = null;
             if (Options.parkingAI && useCar) {
-                ushort parkedVehicleId = citizenData.m_citizen.ToCitizen().m_parkedVehicle;
+                ref Citizen citizen = ref citizenData.m_citizen.ToCitizen();
+                ushort parkedVehicleId = citizen.m_parkedVehicle;
 
```

Comment:
do we need to check if vehicle is valid (has the created flag)?

Reply:
CO don't check for it and I don't see any chance for parked vehicle without Created flag.